onnxruntime/core/providers/dnnl/dnnl_common.h

```diff
@@ -65,9 +66,11 @@ class PrimitivePool {
  private:
   // For thread safety, the map needs to be kept in thread local storage.
   static inline std::unordered_map<std::string, std::unique_ptr<PrimitiveBase>>& GetMap() {
-    static thread_local std::unordered_map<std::string, std::unique_ptr<PrimitiveBase>> map;
-    return map;
+    using MapType = std::unordered_map<std::string, std::unique_ptr<PrimitiveBase>>;
```

Comment:
You may do it in a simpler way:


 std::unordered_map<std::string, std::unique_ptr<PrimitiveBase>> map;

static inline std::unordered_map<std::string, std::unique_ptr<PrimitiveBase>>& GetMap() { 
  return map;
}

Reply:
I'll have to investigate that in the future (was initially using VS 2017, this fix only works in 2019)